SendBird-iOS/OpenChannel/OpenChannelChat/OpenChannelChatViewController.swift

```diff
@@ -585,12 +585,11 @@ class OpenChannelChatViewController: UIViewController, UITableViewDelegate, UITa
                 cell = userMessageCell
             }
         }
-        else if self.messages[indexPath.row] is SBDFileMessage {
-            let fileMessage = self.messages[indexPath.row] as! SBDFileMessage
+        else if let fileMessage = self.messages[indexPath.row] as? SBDFileMessage {
             guard let sender = fileMessage.sender else { return cell }
             guard let fileMessageRequestId = fileMessage.requestId else { return cell }
             guard let currentUser = SBDMain.getCurrentUser() else { return cell }
-
+            
             if sender.userId == currentUser.userId && self.preSendMessages[fileMessageRequestId] != nil {
```

Comment:
It would be better to change `&&` into a comma
```Swift
if sender.userId == currentUser.userId, self.preSendMessages[fileMessageRequestId] != nil { ... }
```

Reply:
I mainly fixed bugs in this PR. I will look into syntax improvement later and PR separately. Thanks to point it out tho :)